sources/lib/util/options.ml

```diff
@@ -26,926 +26,1609 @@
+let vinst_after_bj = ref false
+let vdisable_weaks = ref false
+let vdefault_input_lang = ref ".why"
+let vanswers_with_loc = ref true
+
+let vunsat_mode = ref false
+let vinline_lets = ref false
+
+let vreplay = ref false
+let vreplay_used_context = ref false
+let vreplay_all_used_context = ref false
+let vsave_used_context = ref false
+
+let vprofiling_period = ref 0.
+let vprofiling = ref false
+
+
+(* We don't want to handle functions with more than 10 arguments so
+   we need to split the debug options to gather them in the end.
+   Problems with this way of doing is the options in each "group" are sorted
+   alphabetically before we split the corresponding group. Adding a new one may
+   break the sorting which is why each group contains 7/8/9 options as of now
+   to allow the adding of new ones in the right group
+*)
+
+(* type rules = RParsing | RTyping | RSat | RCC | RArith | RNone *)
+
+(* let rules_of_string = function *)
+(*   | "parsing" -> RParsing *)
+(*   | "typing" -> RTyping *)
+(*   | "sat" -> RSat *)
+(*   | "cc" -> RCC *)
+(*   | "arith" -> RArith *)
+(*   | _ -> RNone *)
+
+type dbg_opt_spl1 = {
+  debug : bool;
+  debug_ac : bool;
+  debug_adt : bool;
+  debug_arith : bool;
+  debug_arrays : bool;
+  debug_bitv : bool;
+  debug_cc : bool;
+  debug_combine : bool;
+  debug_constr : bool;
+}
+
+type dbg_opt_spl2 = {
+  debug_explanations : bool;
+  debug_fm : bool;
+  debug_fpa : int;
+  debug_gc : bool;
+  debug_interpretation : bool;
+  debug_ite : bool;
+  debug_matching : int;
+  debug_sat : bool;
+  debug_sat_simple : bool;
+}
+
+type dbg_opt_spl3 = {
+  debug_split : bool;
+  debug_sum : bool;
+  debug_triggers : bool;
+  debug_types : bool;
+  debug_typing : bool;
+  debug_uf : bool;
+  debug_unsat_core : bool;
+  debug_use : bool;
+  debug_warnings : bool;
+  rules : int;
+}
+
+type dbg_opt = {
+  dbg_opt_spl1 : dbg_opt_spl1;
+  dbg_opt_spl2 : dbg_opt_spl2;
+  dbg_opt_spl3 : dbg_opt_spl3
+}
+
+type case_split_opt =
+  {
+    case_split_policy : Util.case_split_policy;
+    enable_adts_cs : bool;
+    max_split : Numbers.Q.t;
+  }
+
+type context_opt =
+  {
+    replay : bool;
+    replay_all_used_context : bool;
+    replay_used_context : bool;
+    save_used_context : bool;
+  }
+
+type execution_opt =
+  {
+    answers_with_loc : bool;
+    default_input_lang : string;
+    frontend : string;
+    parse_only : bool;
+    parsers : string list;
+    preludes : string list;
+    type_only : bool;
+    type_smt2 : bool;
+  }
+
+type internal_opt =
+  {
+    disable_weaks : bool;
+    enable_assertions : bool;
+    gc_policy : int;
+  }
+
+type limit_opt =
+  {
+    age_bound : int;
+    fm_cross_limit : Numbers.Q.t;
+    interpretation_timelimit : float;
+    steps_bound : int;
+    timelimit : float;
+    timelimit_per_goal : bool;
+  }
+
+type output_opt =
+  {
+    all_models : bool;
+    complete_model : bool;
+    interpretation : int;
+    model : bool;
+    unsat_core : bool;
+    unsat_mode : bool;
```

Comment:
Replace this option with `-o -output-format=VAL` where VAL can be
- native (default)
- smtlib 
- and SZS for futur support of CASC/tptp

Reply:
I don't see what you mean here ;-)